distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> For me, the online distro should result in the same functional setup as the offline distro 

A separate minimal and online. OK, understood. That makes sense.

> I am also thinking about how to best handle the "openhab-demo" feature installation - at the moment, people have to do it manually in the shell, which is definitely not an easy start, so we need to change this.

My preference would go to extending the dashboard a little bit with a default admin/feature installation page. So that a user can launch openhab, click on the URL displayed in the console on startup (we should add that) to go to the dashboard and easily install new features like the demo. Installation itself should of course be triggered using REST services.

> I was wondering about having a config option in the addons.cfg

I would rather start from the concept that once a user has installed openHAB, they would never have to download a new runtime again. They should be able to just update from within openHAB itself. Of course, this will only work for the online runtime. For the offline, you would still have to download. But won't this be the exception rather than the rule?